src/popup/components/Modals/Default.vue

```diff
@@ -23,7 +23,6 @@
     </slot>
 
     <slot />
-
```

Comment:
Why removing this space? It's useful to make the code easier to view and analyse.